Sample-WPF/FaceAPI-WPF-Samples.csproj

```diff
@@ -57,7 +57,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.ProjectOxford.Face, Version=1.2.5.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Besides the line here, other files that specify the version number:

- ClientLibrary/Properties/AssemblyInfo.cs
- Sample-WPF/Properties/AssemblyInfo.cs

Let me know if there is anything wrong I made.